template/packages/backend/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  projects: ["jest-unit.json", "test/jest-e2e.json"],
+}
```

Comment:
Jest comes through! Using the `projects` feature jest can run multiple test configs with one runner which makes it easy to run e2e and unit tests with one command.